src/query.js

```diff
@@ -156,7 +169,7 @@ var objectFunction = function(fields) {
  * Directly produces a FaunaDB Lambda expression as described in the FaunaDB reference
  * documentation.
  *
- * @param {module:query~ExprArg} var_name
+ * @param {module:query~ExprArg} var
```

Comment:
I think this should match the function arg name, doesn't the linter complained about it?